scripts/functions.sh

```diff
@@ -553,18 +553,21 @@ function get_logged_user_object_id {
 
         case "${clientId}" in
             "systemAssignedIdentity")
-                echo " - logged in Azure with System Assigned Identity - ${MSI_ID}"
-                export TF_VAR_logged_user_objectId=$(az identity show --ids ${MSI_ID} --query principalId -o tsv)
-                export ARM_TENANT_ID=$(az identity show --ids ${MSI_ID} --query tenantId -o tsv)
+                computerName=$(az rest --method get --headers Metadata=true --url http://169.254.169.254/metadata/instance?api-version=2020-09-01 | jq -r .compute.name)
```

Comment:
I added a backlog item to later refactor this to take into account multiple clouds and allow for an override as well.